bash_completion.sh

```diff
@@ -18,6 +18,13 @@ if test -n "${ZSH_VERSION:-}" ; then
   fi
 fi
 
+
+function containsElement () 
```

Comment:
what is the point of this method i dont see it being called

Reply:
removed